distribution/openrct2.d.ts

```diff
@@ -1102,20 +1262,38 @@ declare global {
 
     interface RideFreezeRatingArgs extends GameActionArgs {
         ride: number;
-        type: number; // 0: excitement, 1: intensity, 2: nausea
+        /**
+         * - `0`: Excitement
+         * - `1`: Intensity
+         * - `2`: Nausea
+         */
+        type: number;
         value: number;
     }
 
     interface RideSetAppearanceArgs extends GameActionArgs {
         ride: number;
-        type: number; // see RideSetAppearanceType in openrct2/actions/RideSetAppearanceAction.h
-        // value:
-        // - if type is one of the track or vehicle colours: colour
-        // - if type is VehicleColourScheme: 0: all same, 1: per train, 2: per car
-        // - if type is EntranceStyle: entrance style
-        // - if type is SellingItemColourIsRandom: 0: disabled, 1: enabled
+        /** @see RideSetAppearanceType in
+         * [src/openrct2/actions/RideSetAppearanceAction.h](../src/openrct2/actions/RideSetAppearanceAction.h)
```

Comment:
Nitpicky, but seems more consistent with the rest:

```suggestion
        /** 
         * @see RideSetAppearanceType in [src/openrct2/actions/RideSetAppearanceAction.h](../src/openrct2/actions/RideSetAppearanceAction.h)
```